TextEnrichment/Word/DocumentReader.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Aspose.Words;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace TextEnrichment.Word
```

Comment:
The 'Word' namespace is a little bit confusing, maybe WordDocument? or just Document?